### DISCUSSION TITLE
These 6 or more options can make chat more compact

### DISCUSSION THREAD

**Discussion by Treadspear** [SevenTV/Extension#467] [Ideas]
Please will you consider add these option for the chat:

- add/remove hype train
- add/remove top gifted
- add/remove pinned
   - option to show pinned for 30secons
- add/remove "STREAM CHAT" title and "COMMUNITY" button
- add/remove chat button, channel points button, twitch chat settings icon.

Here is a before and after photo of what chat looks like with all these options disabled.

Before:  https://1drv.ms/i/s!Avv6335OlHOmgTuM4nkO564xvIpw?e=OC5xcA [Microsoft OneDrive]
![Untitled](https://user-images.githubusercontent.com/126298418/230256325-55bb8e9a-df4e-49a0-b961-150bf0d7998f.png)

After: https://1drv.ms/i/s!Avv6335OlHOmgTw-eLOU4-EwaL6b?e=1EJR1Z [Microsoft OneDrive]
![Untit4led](https://user-images.githubusercontent.com/126298418/230256321-a3b4cac8-f0ed-4ed8-a8e0-80ea273c9fe2.png)

This would be great for those who have a single monitor but multitask with another window.
Let me know how this may be useful for you.

Thank you

**Comment by Treadspear** [comment]
usually when i have my screen like this i have to use inspect element to remove all this stuff

**Comment by AnatoleAM** [comment]
Definitely yes, there is a lot clutter in chat, which there really should be options to either make more compacted or hide altogether. This isn't top priority for us at the moment, but if someone wishes to make a pull request we'd be more than happy to merge it in. :+1:
